weave-js/src/common/components/elements/ModifiedDropdown.tsx

```diff
@@ -330,23 +330,6 @@ const ModifiedDropdown: FC<ModifiedDropdownProps> = React.memo(
         cursor: 'move',
       };
 
-      const labelText = item.text ?? '';
```

Comment:
For reference: https://github.com/wandb/weave/pull/552/files

![Screenshot 2023-09-28 at 2 25 20 PM](https://github.com/wandb/weave/assets/103448723/7bd9cfdd-f2bd-48e6-ab65-68c917d9a302)